game/src/graphics/player_hp.rs

```diff
@@ -52,3 +54,39 @@ fn update_hp(
         }
     }
 }
+
+fn update_hp_vignette(
+    player_q: Query<&Health, With<Player>>,
+    time: Res<Time>,
+    mut query: Query<&mut VignetteSettings>, 
+) {
+    if let Ok(health) = player_q.get_single() {
+        let health_percentage = health.current as f32 / health.max as f32;
+        if health_percentage <= 0.25 {
+            let bpm = 103.0;
```

Comment:
These let variables could be moved into consts to save on a little computation time.

Reply:
The speed of the throb feels really fast to me, and maybe too transparent? This is just a matter of taste, though.

Reply:
A slower throb might feel better. If I recall, my thought behind the opacity was if it was too high the edge of the vignette became too sharp.